src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsSkinHelper.java

```diff
@@ -110,8 +111,14 @@ private boolean shouldCutPattern(int patternWidth, boolean isLeft) {
   }
 
   private void hideCoveredSkinElements() {
-    adsComparison.hideSlot(AdsContent.WIKIA_BAR, driver);
-    adsComparison.hideSlot(WIKIA_MESSAGE_BUBLE, driver);
+    closeWikiaNotifications();
+    adsComparison.hideSlot(AdsContent.getSlotSelector(AdsContent.WIKIA_BAR), driver);
+  }
+
+  private void closeWikiaNotifications() {
+    if (!driver.findElements(By.cssSelector(WIKIA_MESSAGE_BUBLE)).isEmpty()){
```

Comment:
consider using `public WebElement forElementPresent(By by, boolean failOnTimeout) {` - not sure how because this class doesn't inherit from BasePageObject but maybe you can make use of it somehow with little rearrangement in the code